en/Extending Obsidian/Obsidian URI.md

```diff
@@ -21,140 +30,239 @@ The `action` parameter is the action that you would like to perform. Available a
 
-### Parameters
+### Add content to an existing note
+
+You can add content to an existing note when opening it using the `prepend` or `append` parameters. This is useful for automation workflows where you want to add timestamped entries, append daily logs, or prepend important reminders to existing notes.
+
+#### Parameters
+
+- `prepend` — Content to add to the top of the file. The operation will attempt to merge [[Properties|properties]] if they exist.
+- `append` — Content to add to the end of the file. The operation will attempt to merge [[Properties|properties]] if they exist.
+
+Both parameters accept URI-encoded text. Remember to encode special characters such as newlines (`%0A`), hash symbols (`%23`), and spaces (`%20`).
+
+#### Examples
+
+- `obsidian://open?vault=my%20vault&file=my%20note&append=%0A%23%23%20New%20section`
+
+  Opens the note `my note.md` and appends a newline followed by "## New section" to the end of the file.
+
+- `obsidian://open?vault=my%20vault&file=my%20note&prepend=Important%20note%3A%20`
+
+  Opens the note `my note.md` and adds "Important note: " to the beginning of the file content (after any properties).
 
-- `vault` can be either the vault name or the vault ID[^1].
-- `file` can be either a file name, or a path from the vault root to the specified file. If the file extension is `md`, the extension can be omitted.
-- `path` an absolute file system path to a file.
-  - Using this parameter will override both `vault` and `file`.
-  - This will cause the app to search for the most specific vault which contains the specified file path.
-  - Then the rest of the path replaces the `file` parameter.
-- `prepend` will add to the top of the file and attempt to merge properties.
-- `append` will add to the end of the file and also attempt to merge properties.
+- `obsidian://open?vault=my%20vault&file=daily%20log&append=%0A-%20%5B%5D%20Task%20added%20via%20automation`
+
+  Opens `daily log.md` and appends a newline followed by a task checkbox "- [ ] Task added via automation".
 
 ## Create note
 
-The `new` action, creates a new note in the vault, optionally with some content.
+The `new` action creates a new note in the vault, optionally with some content.
+
+### Parameters
+
+The `new` action uses the [[#Common parameters]] (`vault`, `file`, `path`) plus the following:
+
+- `name` — The file name to be created. If this is specified, the file location will be chosen based on your "Default location for new notes" preferences in [[Settings]].
+- `content` (optional) — The contents of the note. This should be URI-encoded text.
+- `clipboard` (optional) — If set to any value, uses the contents of the clipboard instead of the `content` parameter.
+- `silent` (optional) — If set to any value, creates the note without opening it.
+- `append` (optional) — If set to any value, appends content to an existing file if one exists instead of creating a new file.
```

Comment:
"set to any value" might be confusing -- If set to any value (e.g. `1` or `yes`), ...

Reply:
What does append do if a file doesn't exist? What does `new` do if a file already exists?